guide/voice/audio-resources.md

```diff
@@ -0,0 +1,93 @@
+---
+forceTheme: purple
+---
+
+# Audio Resources
+
+Audio resources contain audio that can be played by an audio player to voice connections.
+
+## Cheat sheet
+
+### Creation
+
+There are many ways to create an audio resource. Below are some example scenarios:
+
+```ts
+const { createAudioResource, StreamType } = require('@discordjs/voice');
+const { createReadStream } = require('fs');
+const { join } = require('path');
+
+// Basic, default options are:
+// Input type is unknown, so will use FFmpeg to convert to Opus under-the-hood
+// Inline volume is opt-in to improve performance
+let resource = createAudioResource(join(__dirname, 'file.mp3'));
+
+// Will use FFmpeg with volume control enabled
+resource = createAudioResource(join(__dirname, 'file.mp3'), { inlineVolume: true });
+resource.volume.setVolume(0.5);
+
+// Will play .ogg or .webm Opus files without FFmpeg for better performance
+// Remember, inline volume is still disabled
+resource = createAudioResource(createReadStream(join(__dirname, 'file.ogg'), {
+	inputType: StreamType.OggOpus
+}));
+
+// Will play with FFmpeg due to inline volume being enabled.
+resource = createAudioResource(createReadStream(join(__dirname, 'file.webm'), {
+	inputType: StreamType.WebmOpus,
+	inlineVolume: true
+}));
+
+player.play(resource);
+```
+
+### Deletion
+
+The underlying streams of an audio resource are destroyed and flushed once an audio player is done playing their audio. Make sure to remove any references that you have created to the resource to prevent memory leaks.
+
+## Handling errors
+
+For most scenarios, you will create an audio resource for immediate use for an audio player. This means that the audio player will propagate errors from the resource for you, so you can attach `error` handlers to the player instead of the resource.
```

Comment:
```suggestion
For most scenarios, you will create an audio resource for immediate use by an audio player. This means that the audio player will propagate errors from the resource for you, so you can attach `error` handlers to the player instead of the resource.
```